homeassistant/components/home_connect/switch.py

```diff
@@ -19,12 +22,45 @@
     BSH_POWER_ON,
     BSH_POWER_STATE,
     DOMAIN,
+    REFRIGERATION_DISPENSER,
+    REFRIGERATION_SUPERMODEFREEZER,
+    REFRIGERATION_SUPERMODEREFRIGERATOR,
 )
-from .entity import HomeConnectEntity
+from .entity import HomeConnectDevice, HomeConnectEntity
 
 _LOGGER = logging.getLogger(__name__)
 
 
+@dataclass(frozen=True, kw_only=True)
+class HomeConnectSwitchEntityDescription(SwitchEntityDescription):
+    """Switch entity description."""
+
+    on_key: str
+    exist_fn: Callable[[HomeConnectDevice], bool]
```

Comment:
I think we can also remove this one for now since all of them are `on_key in ...`